control-plane/p4RuntimeSerializer.cpp

```diff
@@ -216,7 +216,7 @@ externalId(const IR::IDeclaration* declaration) {
             return boost::none;
         }
 
-        const uint32_t id = idConstant->value.get_si();
+        const uint32_t id = static_cast<uint32_t>(idConstant->value);
```

Comment:
No automatic casting?

Reply:
All of boost::multiprecision's conversion operators are marked 'explicit', so you can't implicitly convert to any integer type.  So need the static_cast for a simple conversion.